vendor/modules.txt

```diff
@@ -561,6 +561,7 @@ k8s.io/apimachinery/pkg/util/managedfields
 k8s.io/apimachinery/pkg/util/mergepatch
```

Comment:
Consider formatting this commit message similar to https://github.com/openshift/windows-machine-config-operator/commit/c5dcc3f57afff7ff4f3338c7c85b88f5dcce5fc2.

and adding the commands you ran is helpful too. (e.g. Ran: `my-command.sh`)